src/api/cpp/backend/backend_aux.h

```diff
@@ -19,20 +19,21 @@
 
 #include <mutex>
 #include <string>
-#include "nixl_types.h"
-#include "nixl_descriptors.h"
+
 #include "common/nixl_time.h"
+#include "nixl_descriptors.h"
+#include "nixl_types.h"
 
 // Might be removed to be decided by backend, or changed to high
 // level direction or so.
 typedef std::vector<std::pair<std::string, std::string>> notif_list_t;
 
 
 class nixlBackendOptionalArgs {
-    public:
-        // During postXfer, user might ask for a notification if supported
-        nixl_blob_t notifMsg;
-        bool        hasNotif = false;
+ public:
```

Comment:
I personally prefer `public` and etc. specifiers to be aligned with the `class` and not "one space away"
```suggestion
public:
```

Reply:
Done.